command-expansion-server/src/lib/codegen/CommandTypeCodegen.ts

```diff
@@ -85,65 +84,72 @@ ${doc}
     const minRepeat = repeatArg.repeat?.min ?? 0;
     const maxRepeat = repeatArg.repeat?.max ?? 10;
 
-    const overloadDeclarations: string[] = [];
+    let repeatArgsDeclaration: Array<{ name: string; type: string }>[] = [];
+    let methodParameters: string[] = [];
+    let interfaceParameters: string[] = [];
     let argsOrder: string[] = [];
+
     for (let i = minRepeat; i <= maxRepeat; i++) {
-      // language=TypeScript
-      let repeatArgsDeclaration = '';
-      let repeatArgsNames: string[] = [];
+      let repeatArgs: Array<{ name: string; type: string }> = [];
+
       for (let n = 1; n < i; n++) {
         if (repeatArg.repeat) {
-          repeatArgsNames = repeatArgsNames.concat(repeatArg.repeat?.arguments.map(arg => `"${arg.name}${n}"`));
+          repeatArgs = repeatArgs.concat(
+            repeatArg.repeat.arguments.map(arg => {
+              return { name: `${arg.name}${n}`, type: mapArgumentType(arg, enumMap) };
```

Comment:
This likely doesn't matter too much since this is generated code but a character to more clearly separate the argument name and `n` may be good to have. Maybe:
```ts
${arg.name}_${n}
```

Reply:
Also, this `if (repeatArg.repeat)` can be lifted above the `for (...)`.

Reply:
>This likely doesn't matter too much since this is generated code but a character to more clearly separate the argument name and n may be good to have.

Ohh, looks like the format you have is sticking to some spec. that Clipper expects (see #210), is that right?

Reply:
Let me try lifting the `if` and run some tests to see if that trips up anything. I am not opposed to adding an `_` I don't see it breaking anything but I will have to do some testing.

Reply:
I lifted the `if` and add `_` for the argument name:

ex. 

```
        bundle_name1: VarString<8, 1024>, --> bundle_name_1: VarString<8, 1024>
        number_of_bananas1: U8, -->  number_of_bananas_1 : U8
```

Reply:
Oh sweet, thanks Ryan!